src/Framework/AspNetCoreAnalyzers/src/Analyzers/DiagnosticDescriptors.cs

```diff
@@ -196,4 +196,13 @@ internal static class DiagnosticDescriptors
         DiagnosticSeverity.Warning,
         isEnabledByDefault: true,
         helpLinkUri: "https://aka.ms/aspnet/analyzers");
+
+    internal static readonly DiagnosticDescriptor AtMostOneFromBodyAttribute = new(
+        "ASP0024",
+        "At most one parameter can have a [FromBody] attribute.",
```

Comment:
Localizable strings?

Reply:
Covering this one over and also created an issue to do all the other ones as a separate bulk PR: #46498